.github/workflows/terraform.yml

```diff
@@ -1,6 +1,10 @@
-on:
+on: 
+  push:
```

Comment:
Why do you suggest doing this on push in addition to a PR?

Reply:
Just to make sure that the test will run even if someone bypass the PR

Reply:
I'm sorry bit this sounds a bit like bloat to me.

Reply:
This also enabled me to test the action while i was working on my own fork we should maybe even change it to include all branches so the test would run before someone makes a PR